cg/services/order_validation_service/order_validation_service.py

```diff
@@ -5,5 +5,5 @@
 
 class OrderValidationService(ABC):
     @abstractmethod
-    def validate(self, order_json: str) -> ValidationErrors:
+    def _get_errors(self, order_json: str) -> dict:
```

Comment:
This should be called `validate` (just rename it in this file)

Reply:
Also, it would be great if we returned an actual pydantic object instead of a dict. It makes the type signature of the function a bit mysterious and says nothing about the structure of it. Lets save that for later though.